dali/python/nvidia/dali/plugin/paddle.py

```diff
@@ -222,18 +236,11 @@ def __init__(self,
                 normalized_map[v[0]] = v[1]
         self.normalized_map = normalized_map
 
-        output_map = [isinstance(v, str) and v or v[0] for v in output_map]
-        assert len(set(output_map)) == len(output_map), \
-            "output_map names should be distinct"
-        self.output_map = output_map
-
-        # We need data about the batches (like shape information),
-        # so we need to run a single batch as part of setup to get that info
-        for p in self._pipes:
-            with p._check_api_type_scope(types.PipelineAPIType.ITERATOR):
-                p.schedule_run()
         self._first_batch = None
-        self._first_batch = self.next()
+        try:
+            self._first_batch = self.next()
+        except StopIteration:
+            assert False, "It seems that there is not data in the pipeline, please check if last_batch_policy is set PARTIAL and batch size is bigger than the shard size"
```

Comment:
```suggestion
            assert False, "It seems that there is no data in the pipeline. This may happen if `last_batch_policy` is set to PARTIAL and the requested batch size is greater than the shard size."
```

Reply:
Done